renderdoc/api/replay/data_types.h

```diff
@@ -1360,10 +1404,15 @@ Valid values are 1 (depending on API), 2 or 4, or 0 if the drawcall is not an in
 operation.
 )");
   ResourceId copySource;
+  DOCUMENT(R"(The :class:`Subresource` specifying which part in copySource is used.)");
```

Comment:
It would be nice to update the docstring to :data:\`copySource\` to link up to the other member.

Reply:
np.

Reply:
Should be good now.